clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -77,6 +79,10 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
       if (!description.env) description.env = [:]
       if (!description.labels) description.labels = [:]
 
+      if (description.labels.containsKey(INTERESTING_HEALTH_PROVIDER_NAMES) && "Titus" in description.labels.get(INTERESTING_HEALTH_PROVIDER_NAMES)) {
```

Comment:
Not really sure what this is doing ... seems like it's overriding the `interestingHealthProviderNames` label value? 

I don't believe it's necessary, is it?